README.adoc

```diff
@@ -1,4 +1,72 @@
 = Play module providing a JCache based cache API implementation
 
 This project provides a https://www.playframework.com/documentation/2.5.x/ModuleDirectory[module] for https://www.playframework.com[Play Framework]
-that enables the use of https://github.com/jsr107/jsr107spec[JCache compliant] cache implementations as providers for the https://www.playframework.com/documentation/2.5.x/ScalaCache[Play Cache API].
\ No newline at end of file
+that enables the use of https://github.com/jsr107/jsr107spec[JCache compliant] cache implementations as providers for the https://www.playframework.com/documentation/2.5.x/ScalaCache[Play Cache API].
+
+== Features
+
+This module is intended as a full replacement of the default Play Cache implementation.
+
+It requires a valid JCache implementation on the classpath.
+It also requires a registered extension for `JCacheWrapper` to enable lifting limitations introduced by the JCache API compared to the Play Cache API.
```

Comment:
This reads weird, maybe: "...a registered `JCacheWrapper` extension to enable..."